### PR TITLE
Wallet: illustrations

### LINKED ISSUE #17925: Wallet: Empty tabs illustrations
Add the illustration resources for empty tabs in Wallet.

Designs: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=2224-464224&mode=design&t=jpluM87nZPweMih8-0

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/17925

This PR adds empty state illustrations in wallet.

Demo:

https://github.com/status-im/status-mobile/assets/29354102/5e4e9f2f-65bc-485f-9c9f-a755ce03d1f5

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -765,7 +765,7 @@ SPEC CHECKSUMS:
   RNLanguages: 962e562af0d34ab1958d89bcfdb64fafc37c513e
   RNPermissions: ad71dd4f767ec254f2cd57592fbee02afee75467
   RNReactNativeHapticFeedback: 2566b468cc8d0e7bb2f84b23adc0f4614594d071
-  RNReanimated: 42f56dc5c032a11177b9ea12cdb57285318b432e
+  RNReanimated: 06a228c5a245ef7b5b03f0efc29d76ce4db9031c
```
Comment (J-Son89) [inline ios/Podfile.lock]: revert?

File: src/quo/foundations/resources.cljs [hunk 2]
```diff
@@ -5,7 +5,14 @@
    :keycard-chip-light (js/require "../resources/images/ui2/keycard-chip-light.png")
    :keycard-chip-dark  (js/require "../resources/images/ui2/keycard-chip-dark.png")
    :keycard-watermark  (js/require "../resources/images/ui2/keycard-watermark.png")
-   :bored-ape          (js/require "../resources/images/mock2/bored-ape.png")})
+   :bored-ape          (js/require "../resources/images/mock2/bored-ape.png")
+   :no-funds
+   {:light (js/require "../resources/images/ui2/no-funds-light.png")
```
Comment (J-Son89) [inline src/quo/foundations/resources.cljs:12]: should we make a  separate map for themed images ? just to be safe with the usage?

File: src/status_im2/contexts/wallet/account/view.cljs [hunk 3]
```diff
@@ -25,7 +25,8 @@
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}
-   {:id :permissions :label (i18n/label :t/permissions) :accessibility-label :permissions}
+   ;; Disabled for new wallet MVP
+   ;{:id :permissions :label (i18n/label :t/permissions) :accessibility-label :permissions}
```
Comment (J-Son89) [inline src/status_im2/contexts/wallet/account/view.cljs]: can we just remove this for now instead?

Comment (J-Son89) [reply src/status_im2/contexts/wallet/account/view.cljs]: also cc @briansztamfater @ulisesmac @mmilad75 @tumanov-alex @smohamedjavid @vkjr - removing permissions tab for the moment as requested from product/design. There is some work to do until this is ready 👍

**Review by J-Son89 — APPROVED** [review]

**Review by Francesca-G — APPROVED** [review]
Looks good ✨

**Pull request merged**